exchanges/exchange_test.go

```diff
@@ -1298,7 +1319,6 @@ func TestSetupDefaults(t *testing.T) {
 	// Test websocket support
 	b.Websocket = stream.New()
 	b.Features.Supports.Websocket = true
-	b.SetupDefaults(&cfg)
```

Comment:
Removing this makes this section of the test a duplicate of `TestIsWebsocketEnabled` :)
Will remove it.